src/Suave/Compression.fs

```diff
@@ -78,14 +78,14 @@ module Compression =
     | GZIP ->
       use gzip = new GZipStream(newFileStream, CompressionMode.Compress)
       do! SocketOp.ofTask (fs.CopyToAsync gzip)
-      gzip.Close()
+      gzip.Dispose()
```

Comment:
No, I’m going to rip it out.  The Close wasn’t needed either, which was it was removed from the profiles I suppose.

From: Steffen Forkmann [mailto:notifications@github.com]
Sent: Thursday, November 12, 2015 10:05 AM
To: SuaveIO/suave <suave@noreply.github.com>
Cc: Kevin Ransom <Kevin.Ransom@microsoft.com>
Subject: Re: [suave] Enable build and run on coreclr for suave (#334)


In src/Suave/Compression.fs<https://github.com/SuaveIO/suave/pull/334#discussion_r44691516>:

> @@ -78,14 +78,14 @@ module Compression =

>      | GZIP ->

>        use gzip = new GZipStream(newFileStream, CompressionMode.Compress)

>        do! SocketOp.ofTask (fs.CopyToAsync gzip)

> -      gzip.Close()

> +      gzip.Dispose()

I understand. But in general? Is it needed?

—
Reply to this email directly or view it on GitHub<https://github.com/SuaveIO/suave/pull/334/files#r44691516>.